app/javascript/components/util/EligibilityTooltip.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import ReactTooltip from 'react-tooltip';
+import moment from 'moment-timezone';
+
+class EligibilityTooltip extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  formatTimestamp(timestamp) {
```

Comment:
I'm starting to think we need a `common.js` file or something to store frequently used helper methods like this.

Reply:
Agree, perhaps in utils.

Reply:
Created a ticket for this: [SARAALERT-654](https://tracker.codev.mitre.org/browse/SARAALERT-654)